Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -44,15 +56,28 @@ public class RestRequest {
             }
         }
     }
-
+    
+    /// URLSession delegate for secure loads
+    public func urlSession(_ session: URLSession, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
+        let method = challenge.protectionSpace.authenticationMethod
+        let host = challenge.protectionSpace.host
+        switch (method, host) {
+        case (NSURLAuthenticationMethodServerTrust, "https://localhost:8080"):
```

Comment:
Unless I'm mistaken, I think we need to find a way to enable users to declare the urls that should be trusted rather than defaulting to a specific host and port.

Though much more complicated than we might need, Alamofire allows users to describe the server trust policies for specific urls. See an example [here](https://stackoverflow.com/questions/32553414/alamofire-with-a-self-signed-certificate-servertrustpolicy). Perhaps we want something more along these lines?